Source/Shared/Configuration.swift

```diff
@@ -71,7 +71,7 @@ public struct Configuration {
     /** 
         The `m` parameter of API. Global parameter for session.
     */
-    public var mode            : String = "swarm"
+    public var mode            : String? = "swarm"
```

Comment:
Why do you need it nil? There is only two possible values: "swarm" or "foursquare"